app.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topics.controller");
+
+const {
+  handlePsqlErrors,
+  handleCustomErrors,
+  handleServerErrors,
+} = require("./errors/errors");
+
+const app = express();
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+app.use(handlePsqlErrors);
+app.use(handleCustomErrors);
+app.use(handleServerErrors);
+
```

Comment:
It's important that on PRs for certain tickets we don't include code that for functionality not on the ticket (i.e. not included in Task 2).
There's quite a bit of code here that would be included on other tickets, for example lines 11 (we're not posting so we don't need access to the request body), and line 15 (we don't have any PSQL errors to check for on this ticket). Get rid of any code that isn't needed for this specific ticket please!